ggml/CMakeLists.txt

```diff
@@ -155,6 +155,7 @@ option(GGML_CUDA_GRAPHS                     "ggml: use CUDA graphs (llama.cpp on
 
 option(GGML_HIP                             "ggml: use HIP"                                   OFF)
 option(GGML_HIP_GRAPHS                      "ggml: use HIP graph, experimental, slow"         OFF)
+option(GGML_HIP_NO_VMM                      "ggml: do not try to use HIP VMM"                 ON)
```

Comment:
At some point we should rename all the negative options (e.g. `GGML_CUDA_NO_VMM` -> `GGML_CUDA_VMM`) and set the default value accordingly instead. This is mostly an artifact from the time Makefile was used and didn't have a good way to set defaults.